custom_components/mqtt_vacuum_camera/utils/connection/decompress.py

```
@@ -0,0 +1,338 @@
+                    LOGGER.debug("Cleaned up %d expired cache entries", len(expired_keys))
+            except Exception as e:
+                LOGGER.error("Error in cache cleanup: %s", e)
+
+    async def _worker_loop(self) -> None:
+        """
+        Consume tasks: each is (priority, (topic, payload, data_type, future)).
+        """
+        loop = asyncio.get_running_loop()
+
+        # Pre-compile lambda functions to avoid recreation on each iteration
+        hypfer_decompress: Callable = lambda p: isal_zlib.decompress(p).decode()
```

Ruff (0.8.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `hypfer_decompress` as a `def`

(E731)